### DISCUSSION TITLE
Bluetooth with multiple Controllers

### DISCUSSION THREAD

**Discussion by DwagahnUwU** [nefarius/DsHidMini#246] [Support]
When I connect my DS5 and DS3, the joysticks go at like, two frames a second on the DS3, and I don't know how to fix it. This only happens when connected via Bluetooth.

https://user-images.githubusercontent.com/111663864/185767760-c2a8fc15-05ed-4138-b88e-b00745b83c0d.mp4

.

**Comment by Kanuan** [comment]
Your BT adapter is probably not managing to maintain the necessary data rate between both controllers. I have a DualSense and can use it along the DS3 no problem
